helm/defectdojo/templates/celery-beat-deployment.yaml

```diff
@@ -68,15 +68,11 @@ spec:
       {{- end }}
       {{- if .Values.dbMigrationChecker.enabled }}
```

Comment:
I know that the dbMigrationChecker is enabled by default, but shouldn't be outside this conditional? I think using CloudSQL is not related with the fact that you must have enabled dbMigrationChecker
I tested your MR using an external CloudSQL instance and it only works if dbMigrationChecker is enabled.

Reply:
I agree with out in this point

Reply:
Thanks for the feedback, I solved it in the new commit